config/main.py

```diff
@@ -14,34 +14,48 @@
 import ipaddress
 from swsssdk import ConfigDBConnector, SonicV2Connector, SonicDBConfig
 from minigraph import parse_device_desc_xml
+from config_mgmt import ConfigMgmtDPB
 from utilities_common.intf_filter import parse_interface_in_filter
 
 import aaa
 import mlnx
 import nat
 
+# import port config file path
+from sfputil.main import get_platform_and_hwsku
+from portconfig import get_port_config_file_name
+from portconfig import get_child_ports
+
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help', '-?'])
 
 SONIC_GENERATED_SERVICE_PATH = '/etc/sonic/generated_services.conf'
 SONIC_CFGGEN_PATH = '/usr/local/bin/sonic-cfggen'
 SYSLOG_IDENTIFIER = "config"
+INTF_KEY = "interfaces"
+
+(platform, hwsku) =  get_platform_and_hwsku()
```

Comment:
These variables, (platform, hwsku) are not used anywhere other than on the next line as parameters to `get_port_config_file_name()`. Also, we are not simply setting a constant on the line below, like with the others. What happens if `get_port_config_file_name()` fails? Maybe you could do something similar to what's done in the block on lines 128-132:

```
try:
    version_info = sonic_device_util.get_sonic_version_info()
    asic_type = version_info['asic_type']
except KeyError, TypeError:
    raise click.Abort()
```

Reply:
modified as requested.